metadata.rb

```diff
@@ -6,5 +6,5 @@
 long_description 'Installs/Configures deploy-user'
 version '1.4.2'
 
-depends 'ssh_known_hosts', '~> 0.2'
+depends 'ssh_known_hosts', '~> 4.0'
```

Comment:
there were various issues @shrikeh can prob fill you in on. However, we've found since that although the custom way does /home/deploy/.ssh/known_hosts, we will need it to go back to /etc/ssh/ssh_known_hosts (or whatever the global one is called, as using setfacl for granting permissions to the ssh user rather than sudo for capistrano in some projects (Cap v3 ones for instance)